File: challenges/2-tabuada/python/jvhk/tabuada.py
num = int(input("Digite um número: "))
for x in range(num):
    indice = x + 1
    print("===========================")
    print(f"A tabuada do número {indice} é:" )
    for y in range(10): #printa as multiplicações 1*1 = 1 ...
        acumuladora = y+ 1
        mult = indice * acumuladora
        print(f"  {indice}*{acumuladora} = {mult}") #formato 1*1 = 1  para todos os números
print("===========================")

File: challenges/2-tabuada/java/guilhermeaureliano/Tabuada.java
import java.util.Scanner;

public class Tabuada {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		
		System.out.print("Digite um número: ");
		int x = input.nextInt();
		
		for (int i = 1; i <= x; i++) {
			System.out.println("===========================");
			System.out.println("A tabuada do número " + i + " é:");
			for (int j = 1; j <= 10; j ++) {
				System.out.print(i + "*" + j + " = " );
				System.out.print(i* j + "\n");
			}
		}
		
		input.close();
	}
}
